contracts/infrastructure/WalletFactory.sol

```diff
@@ -48,6 +48,7 @@ contract WalletFactory is Owned, Managed {
     event ModuleRegistryChanged(address addr);
     event RefundAddressChanged(address addr);
     event WalletCreated(address indexed wallet, address indexed owner, address indexed guardian);
+    event WalletFeeRefund(address indexed wallet, address indexed refundAddress, address refundToken, uint256 refundAmount);
```

Comment:
Does refundAddress really need to be indexed? If not we could just add the refund info the the WalletCreated event instead of adding an extra event.

Reply:
Matt agreed to drop the `refundAddress` and use `WalletCreated`so I'll do that.